client/db.go

```diff
@@ -158,6 +158,9 @@ func Open(addr string) (*DB, error) {
 	if err != nil {
 		return nil, err
 	}
+	if u.User == nil {
+		u.User = url.User("root")
```

Comment:
Thanks for catching this.

I'm wondering if `"root"` is the correct default. Other options are the current user's name or to make the user field required and to return an error here. Doesn't matter at the moment since we don't authenticate the user yet, just thinking out loud.